### PR TITLE
docs: added note about adding jupyterhub dependencies

### PR DESCRIPTION
**Pull request by adwk67**
Added note about adding jupyterhub dependencies.

### CODE REVIEW

File: docs/modules/demos/pages/jupyterhub-pyspark-hdfs-anomaly-detection-taxi-data.adoc [hunk 1]
```diff
@@ -136,17 +136,71 @@ You should arrive at your workspace:
 image::jupyterhub-pyspark-hdfs-anomaly-detection-taxi-data/jupyter_hub_workspace.png[]
 
 Now you can double-click on the `notebook` folder on the left, open and run the contained file.
-Click on the double arrow (⏩️) to execute the Python scripts.
+Click on the double arrow (⏩️) to execute the Python scripts (click on the image below to go to the notebook file).
 
-image::jupyterhub-pyspark-hdfs-anomaly-detection-taxi-data/jupyter_hub_run_notebook.png[]
+image::jupyterhub-pyspark-hdfs-anomaly-detection-taxi-data/jupyter_hub_run_notebook.png[link="../../../../stacks/jupyterhub-pyspark-hdfs/notebook.ipynb"]
```
Comment (xeniape) [inline docs/modules/demos/pages/jupyterhub-pyspark-hdfs-anomaly-detection-taxi-data.adoc:141]: The link is not working like this, at least when I generate the docs and try to click it (in the generated documentation the files outside of document files, images, attachments etc. are not present).

```suggestion
image::jupyterhub-pyspark-hdfs-anomaly-detection-taxi-data/jupyter_hub_run_notebook.png[link=https://github.com/stackabletech/demos/blob/main/stacks/jupyterhub-pyspark-hdfs/notebook.ipynb,window=_blank]
```

(Also added the `window=_blank` so it opens in a new window and people don't lose the doc page)

**Review by xeniape — APPROVED** [review]
LGTM, just added some minor nits and pieces

File: docs/modules/demos/pages/signal-processing.adoc [hunk 2]
```diff
@@ -65,6 +65,47 @@ image::signal-processing/notebook.png[]
...
+
+Where `requirements.txt` contains:
+
+[source,console]
+----
+psycopg2-binary==2.9.9
+alibi-detect==0.11.4
+----
+
+NOTE: using a custom image requires access to a repository where the image can be made available.
```
Comment (xeniape) [inline docs/modules/demos/pages/signal-processing.adoc:107]:
```suggestion
NOTE: Using a custom image requires access to a repository where the image can be made available.
```

File: docs/modules/demos/pages/jupyterhub-pyspark-hdfs-anomaly-detection-taxi-data.adoc (hunk 1 above)
Comment (adwk67) [reply docs/modules/demos/pages/jupyterhub-pyspark-hdfs-anomaly-detection-taxi-data.adoc:141]: Thank you! I wasn't sure how best to do that.

File: docs/modules/demos/pages/signal-processing.adoc [hunk 3]
```diff
@@ -65,6 +65,47 @@ image::signal-processing/notebook.png[]
 
 The notebook reads the measurement data in windowed batches using a loop, computes some predictions for each batch and persists the scores in a separate timescale table.
 
+=== Adding libraries
+
+The notebook shows (commented out lines at the top of the first cell) one way of installing python libraries: by using an inline `pip` command.
+There are two ways of doing this:
```
[the remark this answers is not in the document]
Comment (adwk67) [reply docs/modules/demos/pages/signal-processing.adoc:70]: I've removed that sentence as it was a bit redundant anyway: 495824a
Does that read better?

[the remark this answers is not in the document]
Comment (xeniape) [reply docs/modules/demos/pages/signal-processing.adoc:70]: yes thank you :)

**Review by xeniape — APPROVED** [review]

**Pull request merged**